rules/windows/builtin/win_mal_service_installs.yml

```diff
@@ -15,6 +15,10 @@ logsource:
 detection:
     selection:
         EventID: 7045
+    malsvc_wce:
+        ServiceName:
+            - 'WCESERVICE'
+            - 'WCE SERVICE'
```

Comment:
duplicated with [/rules/windows/builtin/win_mal_creddumper.yml](https://github.com/Neo23x0/sigma/blob/d7bd90cb24a9e4761600f7e90b3aa72245d99e28/rules/windows/builtin/win_mal_creddumper.yml).

Reply:
Deduplicated it.